iac-extensions/jvm-framework-config/src/test/resources/checks/HardcodedSecretsCheck/micronaut/HardcodedSecretsCheck.properties

```diff
@@ -0,0 +1,341 @@
+azure.credential.client-certificate.pfx-certificate-password=password
+# Noncompliant@+1
+azure.credential.storage-shared-key.account-key=password
+# Noncompliant@+1
+azure.credential.username-password.password=password
+# Noncompliant@+1
+eureka.client.proxy-password=password
+# Noncompliant@+1
+vault.client.proxy-password=password
+# Noncompliant@+1
+spring.cloud.config.proxy-password=password
+# Noncompliant@+1
+spring.cloud.config.password=password
+# Noncompliant@+1
+consul.client.proxy-username=password
+# Noncompliant@+1
+consul.client.asl-token=password
+# Noncompliant@+1
+infinispan.client.hotrod.security.authentication.password=password
+# Noncompliant@+1
+infinispan.client.hotrod.security.ssl.key-store-password=password
+# Noncompliant@+1
+infinispan.client.hotrod.security.ssl.trust-store-password=password
+# Noncompliant@+1
+javamail.authentication.password=password
+# Noncompliant@+1
+mailjet.api-secret=password
+# Noncompliant@+1
+postmark.api-token=password
+# Noncompliant@+1
+sendgrid.api-key=password
+# Noncompliant@+1
+gcp.credentials.encoded-key=password
+# Noncompliant@+1
+micronaut.jms.activemq.artemis.password=password
+# Noncompliant@+1
+micronaut.jms.activemq.classic.password=password
+# Noncompliant@+1
+mongodb.credential=password
+# Noncompliant@+1
+mongodb.apply-connection-string=password
+# Noncompliant@+1
+mongodb.connection-pool.apply-connection-string=password
+# Noncompliant@+1
+mongodb.cluster.apply-connection-string=password
+# Noncompliant@+1
+mongodb.server.apply-connection-string=password
+# Noncompliant@+1
+mongodb.socket.apply-connection-string=password
+# Noncompliant@+1
+mongodb.ssl.apply-connection-string=password
+# Noncompliant@+1
+mqtt.client.password=password
+# Noncompliant@+1
+mqtt.client.ssl.password=password
+# Noncompliant@+1
+neo4j.password=password
+# Noncompliant@+1
+oci.pass-phrase=password
+# Noncompliant@+1
+oci.passphrase=password
+# Noncompliant@+1
+oci.private-key=password
+# Noncompliant@+1
+micronaut.http.client.micronaut.http.oci-oke.client.proxy-password=password
+# Noncompliant@+1
+oci.config.oke-workload-identity.token=password
+# Noncompliant@+1
+pulsar.authentication-jwt=password
+# Noncompliant@+1
+pulsar.tls-trust-store-password=password
+# Noncompliant@+1
+rabbitmq.password=password
+# Noncompliant@+1
+redis.password=password
+# Noncompliant@+1
+micronaut.security.token.jwt.generator.refresh-token.secret=password
+# Noncompliant@+1
+micronaut.security.token.jwt.claims-validators.openid-idtoken=password
+# Noncompliant@+1
+jasync.client.password=password
+# Noncompliant@+1
+vertx.mysql.client.password=password
+# Noncompliant@+1
+vertx.pg.client.password=password
+# Noncompliant@+1
+tracing.zipkin.http.proxy-password=password
+# Noncompliant@+1
+tracing.jaeger.sender.auth-password=password
+# Noncompliant@+1
+otel.exporter.zipkin.proxy-password=password
+
+
+## Properties with wildcard *
+# Noncompliant@+1
+micronaut.http.services.something.proxy-password=password
```

Comment:
I would suggest the following test cases
```
micronaut.http.services..proxy-password=password
micronaut.http.services.proxy-password=password
```
(Not sure if the first one is parseable though)